selfdrive/test/test_car_models.py

```diff
@@ -281,6 +281,16 @@ def get_route_logs(route_name):
     'enableCamera': True,
     'enableDsu': True,
   },
+  "e6a24be49a6cd46e|2019-10-24--13-59-04": {
+    'carFingerprint': TOYOTA.LEXUS_ES_TSS2,
+    'enableCamera': False,
+    'enableDsu': False,
+  },
+  "e6a24be49a6cd46e|2019-10-26--22-38-02": {
```

Comment:
in this drive, I don't see the radar messages. Weird. Do you know why?

Reply:
Oh I think enableDsu should be True then? (Its a non DSU car)

Reply:
yeah, because it does longitudinal control, so enableDsu is hardcoded true. It should be refactor though.